setup.py

```diff
@@ -30,6 +30,9 @@
             'pytest',
             'ruff',
         ],
+        'upscaler': [
+            'RealESRGAN @ git+https://github.com/ai-forever/Real-ESRGAN.git',
```

Comment:
I missed this part in my last review but I think this can actually go into the `install_requires` section.